static/coffee/flows/controllers.coffee

```diff
@@ -1859,6 +1905,9 @@ NodeEditorController = ($rootScope, $scope, $modalInstance, $timeout, $log, Flow
       $scope.action.msg = {}
     $scope.action.msg[$scope.base_language] = message
 
+    $scope.action.quick_responses = $scope.actions_quick_responses
+    $scope.action.buttons_reply = $scope.actions_buttons_reply
+    
```

Comment:
Only quick replies, and lets use quick_replies here instead of responses for consistency.